### ISSUE TITLE
[BUG] hitomi.la not downloading Artist CG and Game CG

### ISSUE THREAD

**Issue by sixtynicooo** [gan-of-culture/get-sauce#115]
Hi, now I can download almost all the albums found on the hitomi.la site, but I realized that some don't download, I'm reporting my code with the error, I'm waiting for confirmation whether it's a bug or if it's just me from problems

```
get-sauce.exe https://hitomi.la/cg/ump-40--girls%E2%80%99-frontline--no-buta-koubi-italiano-1834710.html
or 
D:\hitomi.la ita\UMP 40 Girls Frontline No Buta Koubi>get-sauce.exe https://hitomi.la/cg/ump-40--girls%E2%80%99-frontline--no-buta-koubi-italiano-1834710.html

output error




panic: runtime error: slice bounds out of range [-1:]

goroutine 1 [running]:
github.com/gan-of-culture/get-sauce/extractors/hitomi.extractData({0xc00029c000?, 0x5a?})
        /home/runner/work/get-sauce/get-sauce/extractors/hitomi/hitomi.go:145 +0x645
github.com/gan-of-culture/get-sauce/extractors/hitomi.(*extractor).Extract(0x1025ba0?, {0xc00005e060?, 0xc00005e068?})
        /home/runner/work/get-sauce/get-sauce/extractors/hitomi/hitomi.go:80 +0xfe
github.com/gan-of-culture/get-sauce/extractors.Extract({0xc00005e060, 0x5a})
        /home/runner/work/get-sauce/get-sauce/extractors/extractors.go:127 +0xb6
main.download({0xc00005e060?, 0x5a?})
        /home/runner/work/get-sauce/get-sauce/main.go:46 +0xb5
main.main()
        /home/runner/work/get-sauce/get-sauce/main.go:115 +0x15d

```

Good afternoon

**Comment by gan-of-culture** [comment]
This is a bug and will be solved with #113. There was no support for Artist CG and Game CG I will add it. Thank you :)

**Comment by sixtynicooo** [comment]
Thanks a lot, it seems to work, congratulations on the speed
